for_other_cantons.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan  3 09:20:09 2022
+
+@author: eduardoaraujo
+"""
+import pandas as pd 
+import numpy as np
+import streamlit as st
+import plotly.graph_objects as go
+from forecast import download_button
+from get_data import get_canton_data  
+from sqlalchemy import create_engine
+engine = create_engine("postgresql://epigraph:epigraph@localhost:5432/epigraphhub")
+
+
+
+#@st.cache
+def get_curve(curve):
+    '''
+    Function to get the cases and hosp data in the database
+    '''
+    df = pd.read_sql_table(f'foph_{curve}', engine, schema = 'switzerland', index_col = 'datum', columns=[ 'geoRegion', 'entries'])
+    df.index = pd.to_datetime(df.index)
+    
+    return df
+
+def plot_cases(canton):
```

Comment:
There is a function with the same name in forecast.py. we should eliminate code duplication, they just make code maintenance much harder.